tools/make/kube.mk

```diff
@@ -138,10 +138,10 @@ kube-demo-undeploy: ## Uninstall the Kubernetes resources installed from the `ma
 #	tools/hack/run-kube-local.sh
 
 .PHONY: conformance
-conformance: create-cluster kube-install-image kube-deploy run-conformance delete-cluster ## Create a kind cluster, deploy EG into it, run Gateway API conformance, and clean up.
+conformance: create-cluster kube-install-image kube-deploy install-eg-addons run-conformance delete-cluster ## Create a kind cluster, deploy EG into it, run Gateway API conformance, and clean up.
```

Comment:
GC(envoy-gateway) refernece EnvoyProxy(proxy-config), that make service `monitoring/otel-collector` is required.

Reply:
is this because we are using the same GC for conformance and e2e ? should we maintain 2 different GatewayClasses to keep each test suite more lightweight ?

Reply:
yes, we can do that later.

Reply:
wait https://github.com/envoyproxy/gateway/pull/5091